src/App.tsx

```diff
@@ -34,62 +35,133 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
+  if (sortType !== SortType.NONE) {
+    visibleGoods.sort((firstItem, secondItem) => {
+      return sortType === SortType.LENGTH
+        ? firstItem.length - secondItem.length
+        : firstItem.localeCompare(secondItem);
+    });
+  }
```

Comment:
Try to use `switch` case when you have a limited cases
```suggestion
  switch (sortType) {
    case SortType.ALPHABET:
      visibleGoods.sort((good1, good2) => good1.localeCompare(good2));
      break;

    case SortType.LENGTH:
      visibleGoods.sort((good1, good2) => good1.length - good2.length);
      break;

    default:
      break;
  }
```